flutter/lib/src/web/web_sentry_js_binding.dart

```diff
@@ -93,6 +93,77 @@ class WebSentryJsBinding implements SentryJsBinding {
       return null;
     }
   }
+
+  Map<String, String>? _cachedFilenameDebugIds;
+  Map<String, String>? get cachedFilenameDebugIds => _cachedFilenameDebugIds;
+  int? _lastKeysCount;
+  Map<String, List<String>>? _parsedStackResults;
+
+  @override
+  Map<String, String>? getFilenameToDebugIdMap() {
+    // 1) Read the debug-ID table once
+    final debugIdMap =
+        _globalThis['_sentryDebugIds'].dartify() as Map<dynamic, dynamic>?;
```

Comment:
Can those keys be anything other than `string` type? If not, would Map<string, dynamic> also work?

Reply:
unfortunately doesn't work, Dart won't parse this if I dont use dynamic for the keys